src/components/TimerModal/TimerPlay.js

```diff
@@ -0,0 +1,9 @@
+export default function TimerPlay({play, setPlay, buttonStyle, divPlay, divPause}) {
+
+    return (
+        <button className={buttonStyle} onClick={() => setPlay(!play)}>
```

Comment:
These classnames are being passed as props but I don't see where you pass it

Reply:
It was used on line 5, but removed now to conform to classname standards (as strings)